web/models/stable_diffusion/cache_objects.py

```diff
@@ -18,8 +18,8 @@
 
 model_config = {
     "v2": "stabilityai/stable-diffusion-2",
-    "v2.1base": "stabilityai/stable-diffusion-2-1-base",
-    "v1.4": "CompVis/stable-diffusion-v1-4",
+    "v2_1base": "stabilityai/stable-diffusion-2-1-base",
```

Comment:
Why not use -v2 and -v2-1-base? It's better to keep the same format for all names.

Reply:
I have used _ as separator everywhere to keep same format. But yes, we can decide on formats having more readability, as they may get exposed to UI as well someday.